ClosedXML/Excel/PivotTables/PivotStyleFormats/XLPivotStyleFormatEnums.cs

```diff
@@ -21,4 +20,24 @@ internal enum XLPivotStyleFormatTarget
         Label,
         Data
     }
+
+    public enum XLPivotAreaValues
+    {
+        None = 0,
+        Normal = 1,
+        Data = 2,
+        All = 3,
+        Origin = 4,
+        Button = 5,
+        TopRight = 6,
+        TopEnd = 7
+    }
+
+    public enum XLPivotTableAxisValues
+    {
+        AxisRow = 0,
```

Comment:
Same here. Unless the particular numbers have some meaning (i.e. they coincide to OpenXML values), it is better to reserve `0` to `NotSet` than use an arbitrary value as a default for non-assigned variables. It protects from rare unpleasant bugs.